docs/faq.md

```diff
@@ -0,0 +1,34 @@
+# Frequently Asked Questions
+
+!!! warning
```

Comment:
`warning` or just `info`/`notice` (or whatever the name for a less strong message box is)?